heartbeat/monitors/active/http/check_test.go

```diff
@@ -125,3 +128,63 @@ func TestCheckBody(t *testing.T) {
 		})
 	}
 }
+
+func TestCheckJson(t *testing.T) {
```

Comment:
These tests are a bit redundant given the python integration tests. However, it's a nicer dev/test cycle when using these. I wrote these first.

I'm +1 to just leave them in, they could be useful in the future.